enet/PSPBUILD

```diff
@@ -1,15 +1,15 @@
 pkgname=enet
-pkgver=1.3.17
-pkgrel=2
+pkgver=1.3.18
+pkgrel=1
```

Comment:
```suggestion
pkgrel=3
```

 bump `pkgrel` up

Reply:
I think we need to reset `pkgrel` back to 1 when updating `pkgver`. Correct me if I'm wrong.

Reply:
Yeah, pkgrel at 1 is correct